### ISSUE TITLE
[TF2] Many doors crush players

### ISSUE THREAD

**Issue by hookyboysb** [ValveSoftware/Source-1-Games#1052]
In the recent update, Badwater was updated so that the RED spawn door near 2nd would crush players who were in the way. It seems like this behavior also made its way to other doors.

Right now it seems to be just exit-only spawn doors (like the one on Badwater). For example, Gullywash is a map where this can easily happen.

I can't get proof of this bug without someone else's help, and it's too late to do that. I'll try to get some proof tomorrow.

**Comment by kisak-valve** [comment]
Hello @hookyboysb, are you still experiencing this issue?

**Comment by Wasmanr2** [comment]
Happens in Degroot keep cp too, where if you jump up to high and hit your head on the gate of the last control point it will crush you.

**Comment by Bradasparky** [comment]
> In the recent update, Badwater was updated so that the RED spawn door near 2nd would crush players who were in the way. It seems like this behavior also made its way to other doors.

The Badwater door in question actually has the opposite problem of players getting stuck in it once it closes upon the 2nd point being captured. For the sake of an explanation on how the doors work and what the real problem with them is, I'm going to refer to the first exit door as the inside door, and the 2nd one as the outside door. The inside door's `Blocking Damage` input is set to 0 so that players don't get crushed when it closes. Instead, there is a `trigger_hurt` between both doors that is activated once a `math_counter` is incremented to 2 by both doors closing. The problem with the outside door is, once a player is stuck on the inside door, the player's bounding box is also within the outside's `trigger_multiple` for opening it. Therefore, the `trigger_hurt` won't activate until the stuck player is killed or disconnects.

![trigger_hurt](https://github.com/ValveSoftware/Source-1-Games/assets/49656562/7baa3379-81c6-429b-a433-b1fb8a9aae92)

> Happens in Degroot keep cp too, where if you jump up to high and hit your head on the gate of the last control point it will crush you.

This is not an example of the door crushing players unintentionally. Degrootkeep has a `trigger_hurt` below the gate door which enables once the door closes for the first time, and is never disabled afterward. This could be it's own issue, but is unrelated.

![degrootkeep_door](https://github.com/ValveSoftware/Source-1-Games/assets/49656562/e1116fb4-01c5-4e44-a0ff-82a86bcf14bf)

In summary, doors crushing players is completely intentional provided that they're `Blocking Damage` input deals enough damage to kill them, and Badwater's door issue is a different issue entirely. If there are cases in maps where a one way exit door has it's `Blocking Damage` value set to 0 and is killing players, then there's reason to make a separate issue per map. 

With that said, this issue can be closed. @kisak-valve

**Comment by kisak-valve** [comment]
Closing per the last comment.
